business/workloads.go

```diff
@@ -170,8 +184,63 @@ func (in *WorkloadService) GetPod(namespace, name string) (*models.Pod, error) {
 	return &pod, nil
 }
 
-func (in *WorkloadService) GetPodLogs(namespace, name string, opts *core_v1.PodLogOptions) (*kubernetes.PodLogs, error) {
-	return in.k8s.GetPodLogs(namespace, name, opts)
+func (in *WorkloadService) getParsedLogs(namespace, name string, opts *core_v1.PodLogOptions) (*PodLog, error) {
+	podLog, err := in.k8s.GetPodLogs(namespace, name, opts)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// Matches an ISO8601 full date
+	timestampRegexp := regexp.MustCompile("^\\d{4}-\\d\\d-\\d\\dT\\d\\d:\\d\\d:\\d\\d(\\.\\d+)?(([+-]\\d\\d:\\d\\d)|Z)?(\\+\\d{4})?")
+	severityRegexp := regexp.MustCompile("(\\s+|^)(INFO|WARN|DEBUG|TRACE)\\s+")
```

Comment:
Perhaps declare these as global vars so we only eval one time.  They can't be constants because you can't have a regexp const, but you can have a var that is basically treated like a constant.